tests/unit/amo/pages/TestCollectionList.js

```diff
@@ -101,10 +114,16 @@ describe(__filename, () => {
     );
 
     fakeDispatch.resetHistory();
+    const errorHandler = createStubErrorHandler();
+    renderComponent({ store, errorHandler });
 
-    renderComponent({ store });
-
-    sinon.assert.notCalled(fakeDispatch);
+    sinon.assert.neverCalledWith(
```

Comment:
As above, because we only care that it's *not* called, you can use the simpler `neverCalledWithMatch`.